packages/hardhat-core/test/internal/solidity/compilation-job.ts

```diff
@@ -1,12 +1,12 @@
 import { assert } from "chai";
 
-import * as taskTypes from "../../../src/builtin-tasks/types";
-import { CompilationJobCreationError } from "../../../src/builtin-tasks/types";
 import {
   createCompilationJobFromFile,
   createCompilationJobsFromConnectedComponent,
 } from "../../../src/internal/solidity/compilation-job";
 import { ResolvedFile } from "../../../src/internal/solidity/resolver";
+import * as taskTypes from "../../../src/types/builtin-tasks";
```

Comment:
Thoesn't the linter complain about this?

Reply:
It doesn't apparently, so it's fine.

Reply:
Nope. I'm not sure why some of these types are imported twice in different ways but the linter doesn't point out anything about these.

Reply:
The linter doesn't complain because I added `{ "allow-namespace-imports": true }` to the `no-duplicate-imports` rule. I had to do that to be able to use some interfaces in a "namespaced" way, like `taskTypes.DependencyGraph`, but importing others like `CompilationJobCreationError` without having to do that.

Maybe that's not necessary anymore? I'm not sure. I don't mind resetting that rule and using one import for everything, but it can make the code uglier in some scenarios (which, again, maybe doesn't happen anymore).